sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/CompilerMain.java

```diff
@@ -222,10 +222,11 @@ CompilerMessages run() throws SQLException {
                     System.out.println("Writing UDF stubs to file " + stubs);
                 protosStream.append("""
 // Compiler-generated file
-// Stubs for user-defined functions
+// This file contains stubs for user-defined functions. Copy these stubs to
```

Comment:
I am not as familiar with the word "stub", could we maybe add "function signature" somewhere, like:

```
// This file contains stubs for the user-defined functions from the SQL program.
// Each stub defines a function name and corresponding signature that must be implemented in `udf.rs`.
// Copy these stubs to `udf.rs` and replace their bodies with the actual implementations.
// For more detailed documentation: https://docs.feldera.com/sql/udf
```